static/js/components/dashboard/FinalExamCard.js

```diff
@@ -0,0 +1,141 @@
+// @flow
+import React from 'react';
+import { Card, CardTitle } from 'react-mdl/lib/Card';
+import IconButton from 'react-mdl/lib/IconButton';
+import _ from 'lodash';
+
+import type { Profile } from '../../flow/profileTypes';
+import type { Program } from '../../flow/programTypes';
+import {
+  PEARSON_PROFILE_ABSENT,
+  PEARSON_PROFILE_SUCCESS,
+  PEARSON_PROFILE_IN_PROGRESS,
+  PEARSON_PROFILE_INVALID,
+  PEARSON_PROFILE_SCHEDULABLE
+} from '../../constants';
+import { getPreferredName, getLocation } from '../../util/util';
+
+const cardWrapper = children => (
+  <Card shadow={0} className="final-exam-card">
+    <div className="card-header">
+      <div>
+        <img className="exam-icon" src="/static/images/exam_icon.png" />
+      </div>
+      <div>
+        <CardTitle>
+          Final Exams
+        </CardTitle>
+        <p>
+          {`You must take a proctored exam for each course. Exams may be taken
+            at any `}
+          <a href="">
+            Pearson test center
+          </a>
+          {`. Before you can take an exam, you have to pay for the course and
+          pass the online work.`}
+        </p>
+      </div>
+    </div>
+    {children}
+  </Card>
+);
+
+const accountCreated = (profile, navigateToProfile) => (
+  <div>
+    <div className="info-box split">
+      <div className="flow"> 
+        Your Pearson Testing account has been created. Your information
+        should match the ID you bring to the test center.
+      </div>
+      <div className="address-info">
+        <div className="address">
+          <span className="name">
+            { getPreferredName(profile) }
+          </span>
+          <span>
+            { _.get(profile, ['address']) }
+          </span>
+          <span>
+            { getLocation(profile) }
+          </span>
+          <span>
+            Phone: { _.get(profile, ['phone_number']) }
+          </span>
+        </div>
+        { editProfileButton(navigateToProfile) }
+      </div>
+    </div>
+    We will notify you when you become eligible to schedule course exams.
```

Comment:
I meant visually the sentence below the gray box seems to jut out a bit. Like I said, it's just a nit and a very small visual one at that.